dlme_airflow/models/collection.py

```diff
@@ -23,3 +23,11 @@ def intermediate_representation_location(self):
     def archive_dir(self):
         archive_root = os.environ.get("ARCHIVE_PATH", "archive")
         return os.path.join(archive_root, self.data_path())
+
+    def datafiles(self):
```

Comment:
Would it make sense to have `datafiles()` take a `format` parameter that defaults to `csv`? The use of the output dictionary elsewhere in the code (see comment below) looks a bit odd?

Reply:
That's not a bad idea. I'll walk through this a bit making sure to default to CSV.